content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -907,6 +907,19 @@ configuration file (default `/etc/cni/net.d`) and ensure that the binary is incl
 bin dir (default `/opt/cni/bin`).
 {{< /note >}}
 
+### kubernetes.io/initial-events-list-blueprint
+
+Type: Annotation
+
+Example: `"k8s.io/initial-events-embedded-list":"eyJraW5kIjoiUG9kTGlzdCIsImFwaVZlcnNpb24iOiJ2MSIsIm1ldGFkYXRhIjp7fSwiaXRlbXMiOm51bGx9Cg=="`
+
+Used on: Bookmark Watch Event
```

Comment:
```suggestion
Used on: Event
```

Reply:
This isn't an annotation used on Events. You find this annotation used on the response to a **watch** or **watchcollection** logical verb.

Reply:
But the code [here](https://github.com/kubernetes/kubernetes/blob/8a39b6062060ffb4fba52f53c011bd00428db194/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L452) says it is added to `bookmark event` 

```go
	// The annotation is added to a "Bookmark" event and is used by clients
	// to guarantee the format consistency when reconstructing
	// the list during WatchList processing.
	InitialEventsListBlueprintAnnotationKey = "kubernetes.io/initial-events-list-blueprint"
)
```

Reply:
Also [here](https://github.com/kubernetes/sig-release/blob/5598ad8e5bfe9a8cc923133d2b5e30262ab63012/releases/release-1.32/release-notes/release-notes-draft.md?plain=1#L83)

Reply:
Yes, but _bookmark events_ are different from Events in the events.k8s.io API group.

Reply:
> This isn't an annotation used on Events. You find this annotation used on the response to a **watch** or **watchcollection** logical verb.

Response to **watch** would be watch events. In code I only see "bookmark event" as the one annotation is used on so do you mean it is used on "watch event"

Reply:
(As I understand it), bookmark events are a kind of watch event. See https://kubernetes.io/docs/reference/using-api/api-concepts/#watch-bookmarks

Reply:
If it wasn't clear, https://github.com/kubernetes/website/pull/49545#discussion_r1928667055 was feedback t @tengqm to correct the suggestion made in https://github.com/kubernetes/website/pull/49545#discussion_r1928255057

Reply:
Right.
The `Used On` value is supposed to be a k8s Kind.
That is, on which kind of k8s object you will find this annotation.
An annotation is always associated with an object of a certain kind.

However, there is no "Bookmark Watch Event" kind, or "Watch Event" kind.

Reply:
You might see this annotation on the response to a GET against a collection of Namespaces, or Secrets, or Nodes. The actual kind of objects in the collection isn't really relevant.

To see this annotation in the watch response, it would have to be a particular kind of GET where you use a _watch list_ to stream the collection back to the client.

Reply:
Okay. I found it.
The serialized form is called metav1.WatchEvent.

```go
type WatchEvent struct {
	Type string `json:"type" protobuf:"bytes,1,opt,name=type"`

	// Object is:
	//  * If Type is Added or Modified: the new state of the object.
	//  * If Type is Deleted: the state of the object immediately before deletion.
	//  * If Type is Error: *Status is recommended; other types may make sense
	//    depending on context.
	Object runtime.RawExtension `json:"object" protobuf:"bytes,2,opt,name=object"`
}
```

This means the annotation is attached to a `WatchEvent` object whose kind
is defined in "k8s.io.apimachinery.pkg.apis.meta.v1".

So writing "WatchEvent" is okay. Just don't write "Watch Event".

Reply:
Maybe adding 
`[WatchEvent](/docs/reference/generated/kubernetes-api/v{{< skew currentVersion />}}/#watchevent-v1-meta)`
link would be helpful.